casepro/pods/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from casepro.pods.views import perform_pod_action, read_pod_data
+
+urlpatterns = (
+    url(r'read/(?P<pk>\d+)/$', read_pod_data, name='read_pod_data'),
```

Comment:
The reasoning behind using an index was that we wanted an auto generated thing, that you wouldn't be able to accidentally assign the same thing to two pods and have things break, and to just have less things to configure in general.

Whether this is a numerical index, or a UUID, or something else doesn't really matter.

The only issue I could think of is if you change the settings and restart the server, while someone still has the webpage open, it could break things until they refresh the page, but we would have the same issue with configurable slugs (if a pod is removed, we would still have issues).

It seemed that having less configuration to worry about was a win here, so we're happy with using the numerical index, although there doesn't seem to be a strong winner.